core/domains/email_manager.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
core/domain not core/domains. Short for "domain layer".

Reply:
Done.